### PR TITLE
Day8

### PR DESCRIPTION
**Pull request by Wiuwiti**

### CODE REVIEW

File: css/Common.css [hunk 1]
```diff
@@ -8,40 +17,39 @@
...
 	height: 35px; 
 	vertical-align: middle;
 	padding-right: 4vw;
 	padding-left: 2vw;
 }
 
-.active-page{
+.navigation-bar-active-page{
 	border-style: solid;
 	border-width: 0px 0px 5px 0px;
```
Comment (s-timofte) [inline css/Common.css:45]: Is there any other way you can write this?

File: css/CommonTheme.css [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
+	border-color: #03a8f3;
+}
 
 /*-----------------   Footer -----------------*/
 
-.footerT{
+.page-footer-theme{
 	color:#828384;
 	background-color: #dfdfdf;
 	text-align: center;
```
Comment (s-timofte) [inline css/CommonTheme.css:24]: I think you are trying to set the way for a later themeing support here, which is nice, but with that in mind would text-align and text-decoration belong here or not?

I also appreciate that you are setting up a naming convention, but you also use separators for the same purpose for example `navigation-bar-active-page` might make a lot more sense like this `navigationBar-activePage` wouldn't you say?

PS: making your IDE replace tabs with spaces might save you some room :)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href ="/css/Common.css">
```
Comment (s-timofte) [inline index.html:7]: If you install less you'll end up havinf just one css inclusion here

**Review by s-timofte — APPROVED** [review]

File: css/CommonTheme.css (hunk 2 above)
Comment (Wiuwiti) [reply css/CommonTheme.css:24]: Yes it seems an issue the naming the way I do as soon as I hit more than 3 words. Wanted to firmly respect a naming convention that is why stood so much with the '-' and didn't want to include camel cases. 
Will try to think of a way to remake the naming for the more complex CSS structures

**Pull request merged**